src/app/[locale]/(business)/(with-header)/breweries/[brewerySlug]/beers/[beerSlug]/page.tsx

```
@@ -4,12 +4,19 @@ import { getLocale, getTranslations } from "next-intl/server";
+  addToCellarAction,
+  createStorageLocationAction,
+} from "@/app/[locale]/(business)/(with-header)/users/[username]/cellar/actions";
+import AddToCellarModal from "@/app/_components/add-to-cellar-modal";
 import ShareButton from "@/app/_components/share-button";
 import Button from "@/app/_components/ui/button";
 import { getBeerBySlug } from "@/domain/beers";
+import { getStorageLocationsByUser } from "@/domain/storage-locations";
 import { config } from "@/lib/config";
 import { publicConfig } from "@/lib/config/client-config";
 import { StaticGenerationMode } from "@/lib/config/types";
+import { getCurrentUser } from "@/lib/auth";
```

ESLint

[error] `@/lib/auth` import should occur before import of `@/lib/config`

(import/order)